lib/usd/ui/layerEditor/layerTreeItemDelegate.h

```diff
@@ -124,8 +124,8 @@ class LayerTreeItemDelegate : public QStyledItemDelegate
     const int    CHECK_MARK_AREA_WIDTH = DPIScale(28);
     const int    TEXT_LEFT_OFFSET = (ARROW_AREA_WIDTH + CHECK_MARK_AREA_WIDTH) + DPIScale(2);
     const int    HIGHLIGHTED_FILL_OFFSET = DPIScale(0);
-    const double DISABLED_OPACITY = 0.8;
-    const double HOVER_OPACITY = 0.55;
+    const double DISABLED_OPACITY = 0.4;
+    const double HOVER_OPACITY = 0.4;
```

Comment:
I kept the hover opacity as a separate variable as there is a chance that it becomes 20% based on design direction.